kod/util/factgame/territry.kod

```diff
@@ -564,7 +564,13 @@ messages:
                oFlagpole = send(oRoom,@FindHoldingActive,#class=&FlagPole);
                if send(oFlagPole,@GetFaction) <> (iFaction-1)
                {
-                  send(oFlagPole,@SetFaction,#faction=(iFaction-1));
+                  Send(oFlagPole,@SetFaction,#faction=(iFaction-1));
```

Comment:
Call AdminSetFaction instead of duplicating the code?

Reply:
Calling a function labeled for Admin use within the standard context of the territory game code seemed dangerous to me. I had a tiny Andrew sitting on my shoulder whispering, "What if someone updates AdminSetFaction to perform additional admin code?"